src/ipc-handlers.ts

```diff
@@ -54,6 +54,10 @@ import { SiteServer, createSiteWorkingDirectory } from 'src/site-server';
 import { DEFAULT_SITE_PATH, getResourcesPath, getSiteThumbnailPath } from 'src/storage/paths';
 import { loadUserData, saveUserData } from 'src/storage/user-data';
 import { DEFAULT_PHP_VERSION } from 'vendor/wp-now/src/constants';
+import {
+	generateSiteCertificate,
+	openCertificate as openCertificateDialog,
```

Comment:
```suggestion
	openCertificate,
```
Seems like a leftover

Reply:
This is not possible because we expose a function with the same name.

Reply:
Ah. Then let's call it `openCertificateInFolder` or something. `openCertificateDialog` is not accurate.